File: app/src/main/java/android1601/itstep/org/kidsgame/program/enums/SECTIONS_TYPE.java
package android1601.itstep.org.kidsgame.program.enums;

/**
 * Created by roman on 23.04.2017.
 */

public enum SECTIONS_TYPE {
    TOYS(1),CLOTHES(2),UNKNOWN(0);
    private int section;

    SECTIONS_TYPE(int section){this.section = section;}

    public static SECTIONS_TYPE sections_type(int value){
        for (SECTIONS_TYPE type: SECTIONS_TYPE.values()) {
            if (type.getPosition()==value){
                return type;
            }
        }
        return UNKNOWN;
    }

    public int getPosition() {
        return section;
    }
}

File: app/src/main/java/android1601/itstep/org/kidsgame/program/enums/SECTIONS_TYPE.kt
package android1601.itstep.org.kidsgame.program.enums

/**
 * Created by roman on 23.04.2017.
 */

enum class SECTIONS_TYPE private constructor(val position: Int) {
    TOYS(1), CLOTHES(2), UNKNOWN(0);


    companion object {

        fun sections_type(value: Int): SECTIONS_TYPE {
            for (type in SECTIONS_TYPE.values()) {
                if (type.position == value) {
                    return type
                }
            }
            return UNKNOWN
        }
    }
}
